doc/contributing/pull-requests.md

```diff
@@ -523,6 +524,25 @@ to fail on specific platforms or for so-called "flaky" tests to fail ("be red").
 It is vital to visually inspect the results of all failed ("red") tests to
 determine whether the failure was caused by the changes in the pull request.
 
+## Managing stale pull requests
+
+Pull requests are a valuable contribution to the project and the collaborators
+work hard to make sure pull requests land as quickly as possible.
+
+Unfortunately not all pull requests land in a timely manner and after an
+extended amount of time without active progress the likelyhood of a pull
+request landing decreases significantly.
+
+For pull requests that are older than a year and for which there have been
+no updates for 5 months, the project uses a stale action to comment on the
+pull request and to mark it as stale. If there are no further updates on
+the pull request in the next month an additional comment is added and
+the pull request is closed.
+
+A simple comment on a pull request can be used to reset a pull request
+marked as stale and prevent it from being a candidate for closure for
+another 6 months.
```

Comment:
I think all of this should be omitted from the contributing docs. We don't need to tell new contributors about our stale PR processing. And the stale PR action should indicate that a comment will keep the PR open, so we don't need to say it here.

Reply:
ok will remove

Reply:
@Trott removed